interactions/api/gateway/client.py

```diff
@@ -392,6 +392,27 @@ def _dispatch_event(self, event: str, data: dict) -> None:
         elif event != "TYPING_START":
             name: str = event.lower()
             try:
+
+                def __modify_guild_cache():
+                    if not (
+                        guild_id := data.get("guild_id")
+                        and not isinstance(obj, Guild)
+                        and "message" not in name
+                    ):
+                        return
+                    if guild := self._http.cache[Guild].get(Snowflake(guild_id)):
+                        model_name = model.__name__.lower()
```

Comment:
Was the model variable accessable from the function? In other words, have you tested it after making it a function? Just in case

Reply:
No, I forgot about that :sweat_smile:

Reply:
Ping me when this is resolved

Reply:
@Toricane done